templates/plausible/meta.yaml

```diff
@@ -36,6 +36,11 @@ schema:
     - appServiceImage
     - databaseServiceName
     - clickhouseServiceName
+    - mailerEmail
```

Comment:
should these be required? someone could use/test plausible without configuring smtp